components/inbound-endpoints/org.wso2.carbon.inbound.endpoint/src/main/java/org/wso2/carbon/inbound/endpoint/protocol/kafka/AbstractKafkaMessageListener.java

```diff
@@ -31,7 +32,7 @@
     protected ConsumerConnector consumerConnector;
     protected InjectHandler injectHandler;
     protected Properties kafkaProperties;
-    protected ConsumerIterator<byte[], byte[]> consumerIte;
+    protected ArrayList<ConsumerIterator<byte[], byte[]>> consumerIte;
```

Comment:
Best way is to define in more generic way like List<ConsumerIterator<byte[], byte[]>> consumerIte